src/main/java/org/ihtsdo/buildcloud/core/dao/ProductDAOImpl.java

```diff
@@ -94,4 +95,15 @@ public Product find(String releaseCenterKey,
 		query.setParameter("productBusinessKey", productKey);
 		return (Product) query.uniqueResult();
 	}
+
+	@Override
+	public Product constructProduct(String releaseCenterKey, String productKey) {
+		// use this instead of fetching from mysql for srs worker
```

Comment:
@astro-snail  It might not be the correct place to have this method here.

Reply:
@CoderMChu I was not sure where to place it. Any suggestion? Thanks!